test/glific/interactive_templates_test.exs

```diff
@@ -318,4 +366,25 @@ defmodule Glific.InteractiveTemplatesTest do
     assert Enum.any?(translations["2"]["options"], fn option -> option["title"] == "परीक्षण 1" end)
     assert Enum.any?(translations["2"]["options"], fn option -> option["title"] == "परीक्षण 2" end)
   end
+
+  test "export the interactive template",
+       %{organization_id: _organization_id} = attrs do
+    interactive =
+      Fixtures.interactive_fixture(Map.merge(@valid_more_attrs, attrs))
+
+    interactive_id = interactive.id
+
+    expected_export_data = """
+    id,Attribute,en,hi
+    #{interactive_id},Header,Quick Reply Test Text 2,त्वरित उत्तर स्थिरता
+    #{interactive_id},Text,How was your experience with Glific?,ग्लिफ़िक त्वरित उत्तर का परीक्षण करें?
+    #{interactive_id},OptionTitle 1,Great,उत्कृष्ट
+    #{interactive_id},OptionTitle 2,Awesome,शानदार
+    """
+
+    {:ok, %{export_data: export_data}} =
+      InteractiveTemplates.export_interactive_template(interactive)
+
+    assert String.trim(export_data) == String.trim(expected_export_data)
+  end
```

Comment:
Do we have a test where content with caption field, if not can we add that too, since we have some extra code regarding that.